patches/tModLoader/Terraria/ModLoader/ModKeybind.cs

```diff
@@ -11,13 +12,16 @@ namespace Terraria.ModLoader
 	public class ModKeybind
 	{
 		internal readonly Mod mod;
-		internal readonly string name; // name from modder: "Random Buff"
-		internal readonly string uniqueName; // eg: "Example Mod: Random Buff" (currently also display name)
+		internal readonly string name; // name from modder: "RandomBuff"
+		internal readonly string uniqueName; // name saved to disk: "ExampleMod: RandomBuff"
 		internal readonly string defaultBinding; // from mod.Load
+		internal readonly string displayNameKey; // Mods.ExampleMod.Keybind.RandomBuff
+		internal string DisplayName => Language.Exists(displayNameKey) ? Language.GetTextValue(displayNameKey) : name; // "Random Buff"
```

Comment:
Given these are all `internal`, they should be safe to change. Can we make them consistent with other `ModType` classes?

Eg, `Mod`, `Name`, `FullName`, `ModTranslation DisplayName`

`DisplayName = LocalizationLoader.GetOrCreateTranslation(Mod, $"Keybind.{Name}");`